cluster-autoscaler/simulator/cluster_snapshot_test.go

```diff
@@ -464,3 +463,102 @@ func TestNodeAlreadyExists(t *testing.T) {
 		}
 	}
 }
+
+func TestPVCUsedByPods(t *testing.T) {
```

Comment:
Please make this test table-based and include a test case with a non-PVC volume.

Reply:
Included the test.
I don't think we should try to make this a table format. Since each case is testing using different setup. Some require add pod/ some delete pod.

Reply:
Each test case is following rougly the same pattern: 
- Add a node
- Add pods to the node
- Do a check
- (optionally) Remove some pods and do a second check

This can be encoded into a table format quite easily.

Reply:
done